.github/workflows/accelsim.yml

```diff
@@ -16,7 +16,7 @@ on:
 # By default regress against accel-sim's dev branch
 env:
   ACCELSIM_REPO: https://github.com/purdue-aalp/accel-sim-framework-public.git
-  ACCELSIM_BRANCH: dev
+  ACCELSIM_BRANCH: dev-tlb
```

Comment:
Do we need the dev-tlb branch of accel-sim for gpgpu-sim to work?
If the config is not configured to use tlbs, then does it still need this branch?

Reply:
I think this is a circular dependency issue: the tlb version of Accel-Sim needs the tlb version of gpgpu-sim and vice versa. We should have some ways to deal with this. Some ideas I have:
- Specify the branch to use for CI runs? Not sure if github allows inputs from users.
- First use the `dev` branch to test `Accel-Sim/dev` with `GPGPU-Sim/dev-tlb`, make sure things are not broken. Then add additional CI test for `Accel-Sim/dev-tlb` and `GPGPU-Sim/dev-tlb`

Also, is it possible to control whether to enable TLB functionality or not?